web_external/Datasets/applyMetadataPage.jade

```diff
@@ -0,0 +1,25 @@
+.isic-form-page.isic-apply-metadata-container
+  h2.isic-page-title Apply metadata
+
+  .isic-form-container
+    include datasetInfoWidget
+    .isic-form-group
+      h3 Select metadata file
+      p.
+        Select the metadata file to apply:
+      .form-group#isic-apply-metadata-select-file-container
+      button#isic-apply-metadata-download-button.btn.btn-default.btn-md(disabled)
+        i.icon-download
+        |  Download
+    .isic-form-group
+      h3 Validate metadata
+      p.
+        This section displays any problems with the metadata.
+        Press the button below to validate the metadata.
+        Once validation succeeds, you will be given an option to save the metadata.
+      .isic-apply-metadata-validate-button-container.text-center
+        button#isic-apply-metadata-validate-button.btn.btn-primary.btn-md(disabled) Validate
+      .isic-apply-metadata-validation-container
```

Comment:
Why is it necessary to have two levels of `<div>` here? Couldn't we just put the `#isic-apply-metadata-validation-container` at the upper level, and dispense with the `.isic-apply-metadata-validation-container` entirely?